presto-docs/src/main/sphinx/release/release-0.210.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.210
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when aliasing columns of tables containing hidden columns (:issue:`11385`).
+* Fix correctness issue when ``GROUP BY DISTINCT`` terms contain references to the same column
+  using different syntactic forms (:issue:`11120`).
+* Fix failures when querying information_schema tables using capitalized names.
+* Remove min/max intersection heuristics from equality stats calculator.
```

Comment:
If this has any user impact, explain it in those terms, otherwise remove

Reply:
cc @arhimondr